en/docs/learn/configuring-a-service-provider-for-adaptive-authentication.md

```diff
@@ -17,6 +17,24 @@ This tutorial guides you through setting up a sample application to try
 out adaptive authentication and configuring it as a service provider in
 WSO2 IS.
 
+!!! Warning "Before you proceed"
+
+    Adaptive Authentication is disabled by default.
+    You need to run **adaptive.sh** (**adaptive.bat** for Windows) in 
+    `          [IS-HOME]/bin         `
+
+    To enable adaptive authentication please proceed following instructions.
+
+    1.   Stop the server if running
+    2.   Run adaptive.sh (adaptive.bat for Windows) (eg: `sh adaptive.sh`)
+    3.   Restart the server
+ 
+    To disable adaptive authentication please proceed following instructions.
```

Comment:
shall we add this at the end of the doc?

Reply:
may I know the reason why? ... if we add at the end of it, there may 2 issues

1. Developer may miss it
2. Developers need to restart after he/she create app and configure

Reply:
The title says as `Before you begin`